style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap");
```

Comment:
What's the other way of installing 3rd party fonts ??

Reply:
```@font-face``` Rule as far as i know i am studying about other methods too